src/libutil/util.cc

```diff
@@ -1093,6 +1093,14 @@ std::pair<int, std::string> runProgram(RunOptions && options)
     StringSink sink;
     options.standardOut = &sink;
 
+    int status = runProgramWithoutCapturingStdout(options);
+
+    return {status, std::move(sink.s)};
+}
+
+// Output = error code
+int runProgramWithoutCapturingStdout(const RunOptions & options)
```

Comment:
I'm not a big fan of this name, and would rather have the "no capture" function be just plain `runProgram` and name the capturing version something else, but that's a more invasive change, so I'll defer to the maintainers/reviewers here on what they prefer.